install/kubernetes/cilium/templates/cilium-nodeinit/daemonset.yaml

```diff
@@ -61,8 +61,42 @@ spec:
           imagePullPolicy: {{ .Values.nodeinit.image.pullPolicy }}
           securityContext:
             privileged: true
-{{- if .Values.nodeinit.revertReconfigureKubelet }}
           lifecycle:
+{{- if and (.Values.eni.enabled) (not .Values.enableIPv4Masquerade) }}
+            postStart:
+              exec:
+                command:
+                  - "nsenter"
+                  - "-t"
+                  - "1"
+                  - "-m"
+                  - "--"
+                  - "/bin/sh"
+                  - "-c"
+                  - |
```

Comment:
I don't think we need to run with nsenter in this case because we have access to the iptables. Let's just make sure we mount this into node-init as well: https://github.com/cilium/cilium/blob/8c49b836cb101b897be95c06410125fb59cd18b5/install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml#L531-L534